docs/en/observability/monitor-logs.asciidoc

```diff
@@ -1,5 +1,5 @@
 [[monitor-logs]]
-= Monitor logs
+= Log monitoring
```

Comment:
I'll note that I expected "Logs monitoring" to be more popular, but nope -- "Log monitoring" by a mile.